### DISCUSSION TITLE
anomalib train problem

### DISCUSSION THREAD

**Discussion by m11113040** [open-edge-platform/anomalib#1702] [Q&A] [answered]
I am new in AI training
I want to train myself data in Anomalib,and I want to use padim model,but i dont know to set the dataset,is there any example

1.  conda activate anomalib_env
2.  install python python=3.10
3.  git clone https://github.com/openvinotoolkit/anomalib.git   #version: v1
4.  cd anomalib
5.  pip install -e .

**Comment by samet-akcay** [comment]
Hi @m11113040, which dataset do you want to use? 

You could refer to these jupyter notebooks:
https://github.com/openvinotoolkit/anomalib/tree/main/notebooks/100_datamodules

or, there is also a section in the documentation showing how to train anomalib models on a custom dataset.
https://anomalib.readthedocs.io/en/latest/markdown/guides/how_to/data/custom_data.html

**Comment by m11113040** [comment]
![image](https://github.com/openvinotoolkit/anomalib/assets/157609224/f35a9868-d578-4ed2-919a-87a4eb10609a)
thank you sir,I take a look this example,and change the path,after that i run (anomalib train --data classification.yaml --model anomalib.models.padim --task CLASSIFICATION) in CLI,but it shows some error

my dataset:
![image](https://github.com/openvinotoolkit/anomalib/assets/157609224/0b92fd49-70e8-4de4-9e48-7a236e319e67)


my error:
![image](https://github.com/openvinotoolkit/anomalib/assets/157609224/b4e27bea-dd02-47db-9958-90a5e10cf21f)

**Reply by willyfh** [reply]
Hi @m11113040, based on your screenshot, it seems you placed the classification.yaml in `src/configs/model/classification.yaml`. Therefore, I suggest updating your argument in the command to `--data src/configs/model/classification.yaml`. Make sure to provide the correct path to the file.

**Reply by m11113040** [reply]
thanks sir,I change my path,and i run  (anomalib train --data classification.yaml --model anomalib.models.padim --task CLASSIFICATION) still have error
![image](https://github.com/openvinotoolkit/anomalib/assets/157609224/b4460b38-5e2b-4805-976e-c6d2086777d1)

**Reply by samet-akcay** [reply]
ideally, it should not ask you openvino stuff, we could fix it later, but can you install openvino for now?

```bash
pip install openvino-dev
```

**Reply by m11113040** [reply]
i re-install openvino but still error
![image](https://github.com/openvinotoolkit/anomalib/assets/157609224/a9b577c3-6755-42df-8c37-8bc406b40bfd)
![image](https://github.com/openvinotoolkit/anomalib/assets/157609224/50ed9bb3-54e6-4f47-bcff-fc78e931546c)

**Reply by m11113040** [reply, marked as answer]
After i use `pip install openvino-dev` will fix  No module named 'openvino.tools.mo'

**Reply by zhaozzt** [reply]
![3](https://github.com/openvinotoolkit/anomalib/assets/127210381/6c2a695d-b1d3-40a7-88dd-2f2edc6b4694)
ME TOO~ help me

**Comment by zhaozzt** [comment]
I encountered the same problem，hhelp me
![1](https://github.com/openvinotoolkit/anomalib/assets/127210381/c8f91464-cc29-4fcb-b632-232c02513992)
